src/test/java/edu/iis/powp/gui/TestPlotSoftPatterns.java

```diff
@@ -32,6 +34,12 @@ private static void setupPresetTests(Context context) {
 	    SelectTestFigureOptionListener selectTestFigureOptionListener = new SelectTestFigureOptionListener();
 		
 		context.addTest("Figure Joe 1", selectTestFigureOptionListener);
+        context.addTest("Figure Joe 2", new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                FiguresJoe.figureScript2(Application.getComponent(DriverManager.class).getCurrentPlotter());
+            }
+        });
```

Comment:
It is good to keep a convention used in the original code (normal class instead of anonymous)